src/screens/Wallet.js

```diff
@@ -75,61 +73,108 @@ class Wallet extends React.Component {
     successMessage: '',
     hasTokenSignature: false,
     shouldShowAdministrativeTab: false,
+    totalSupply: null,
+    canMint: false,
+    canMelt: false,
+    transactionsCount: null,
+    mintCount: null,
+    meltCount: null,
   };
 
-  // Reference for the TokenGeneralInfo component
-  generalInfoRef = React.createRef();
-
-  // Reference for the TokenAdministrative component
-  administrativeRef = React.createRef();
-
   // Reference for the unregister confirm modal
   unregisterModalRef = React.createRef();
 
-  componentDidMount = () => {
+  componentDidMount = async () => {
     this.setState({
       backupDone: hathorLib.wallet.isBackupDone()
     });
 
-    $('#wallet-div').on('show.bs.tab', 'a[data-toggle="tab"]', (e) => {
-      // On tab show we reload token data from the server for each component
-      if (e.target.id === 'token-tab') {
-        this.generalInfoRef.current.updateTokenInfo();
-      } else if (e.target.id === 'administrative-tab') {
-        this.administrativeRef.current.updateTokenInfo();
-      }
-    });
+    await this.updateTokenInfo(this.props.selectedToken);
```

Comment:
We shouldn't execute this request and the next one if it's HTR.

Reply:
You are right, fixed in 9aa96889c9eadbbbb2bc9515f39e00206731267d